jsinterop-ts-defs-processor/src/test/java/com/vertispan/tsdefs/types/varargs/JsTypeWithVarargsMethod.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2023 Vertispan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.vertispan.tsdefs.types.varargs;
+
+import jsinterop.annotations.JsType;
+
+@JsType
+public class JsTypeWithVarargsMethod {
```

Comment:
change this test so we validate it? before this patch it wasn't failing, it was just emitting the wrong .d.ts, as if it was a plain array

Reply:
We will do the test when we finish the dtslint work in which we can write ts tests to verify the output from this class, the java tests are only limited to the failing scenarios that produce a compilation error.